### PR TITLE
add blog post about new binary CLIs

### PR DESCRIPTION
**Pull request by tamarrow**

### CODE REVIEW

**Comment by joel-hamill** [comment]
@tamarrow should this be merged?

**Comment by tamarrow** [comment]
yes!

**Comment by joel-hamill** [comment]
retest this please

**Comment by joel-hamill** [comment]
merging, with errors. the PR has only 1 file, which introduces no errors.

File: src/blog/2016-10-07_binarycli.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: DC/OS CLI, Installation Made Simple
```
Comment (joel-hamill) [inline src/blog/2016-10-07_binarycli.md]: i'll update to say 12/19

**Pull request merged**